doc/radosgw/multisite.rst

```diff
@@ -0,0 +1,487 @@
+                        "name": "us-east",
+                        "endpoints": [
+                            "http:\/\/rgw1:80"
+                        ],
+                        "log_meta": "true",
+                        "log_data": "false",
+                        "bucket_index_max_shards": 0,
+                        "read_only": "false"
+                    }
+                ],
+                "placement_targets": [
+                    {
+                        "name": "default-placement",
+                        "tags": []
+                    }
+                ],
+                "default_placement": "default-placement",
+                "realm_id": "4a367026-bd8f-40ee-b486-8212482ddcd7"
+            }
+        ],
+        "short_zone_ids": [
+            {
+                "key": "83859a9a-9901-4f00-aa6d-285c777e10f0",
+                "val": 630926044
+            }
+        ]
+    },
+    "master_zonegroup": "d4018b8d-8c0d-4072-8919-608726fa369e",
+    "master_zone": "83859a9a-9901-4f00-aa6d-285c777e10f0",
+    "period_config": {
+        "bucket_quota": {
+            "enabled": false,
+            "max_size_kb": -1,
+            "max_objects": -1
+        },
+        "user_quota": {
+            "enabled": false,
+            "max_size_kb": -1,
+            "max_objects": -1
+        }
+    },
+    "realm_id": "4a367026-bd8f-40ee-b486-8212482ddcd7",
+    "realm_name": "gold",
+    "realm_epoch": 2
+    }
+
+
+Starting the radosgw
+--------------------
+
+Before starting the radosgw, the rgw zone and port options need to be mentioned
+in the configuration file. For more details refer to the `Install Ceph Gateway`_
+section of the guide. The configuration section for radosgw should resemble::
+
+  [client.rgw.us-east-1]
+  rgw_frontends="civetweb port=80"
+  rgw_zone=us-east
+
+And start the Ceph Object gateway (according to the OS installation) ::
+
+  sudo systemctl start ceph-radosgw.service
+
+
+Configuring the Secondary zone
+==============================
+
+The following steps will be performed on the node hosting the secondary zone.
+You would've to delete the default zonegroup as mentioned for the primary zone.
+
+Realm configuration
+-------------------
+
+Since a realm was already configured from the first gateway, we pull and make
+that realm the default here::
+
+  # radosgw-admin realm pull --url=http://rgw1:80
+  --access-key=$SYSTEM_ACCESS_KEY --secret-key=$SYSTEM_SECRET_KEY
+  {
+    "id": "4a367026-bd8f-40ee-b486-8212482ddcd7",
+    "name": "gold",
+    "current_period": "b5e4d3ec-2a62-4746-b479-4b2bc14b27d1",
+    "epoch": 2
+  }
+
+  # radosgw-admin realm default --rgw-realm=gold
+
+Secondary Zone Configuration
+----------------------------
+
+We first set the default zonegroup to the created ``us`` zonegroup::
+
+  # radosgw-admin zonegroup default --rgw-zonegroup=us
+
+Next we create the new zone, ``us-west``, with the same system keys::
```

Comment:
@theanalyst, after discussing it with @cbodley:
If we have two different clusters:
on the master: realm creation, zonegroup creation, master zone creation, period update --commit
on the second zone: realm pull, period pull, create second zone, period update --comitt
For one cluster we can have a simpler way of configuration as everything can be done on the master gateway including the second zone creation (no need for realm pull and period pull).